relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -723,15 +746,15 @@ mod tests {
 
     span_description_test!(
         resource_img_comma_with_extension,
-        "https://example.org/p/fit=cover,width=150,height=150,format=auto,quality=90/media/photosV2/weird-stuff-123-234-456.jpg",
-        "resource.img",
-        "https://example.org/*/media/*/weird-stuff-*-*-*.jpg"
+        "https://example.org/p/fit=cover,width=150,height=150,format=auto,quality=90/media/photosV2/weird-stuff-123-234-456.js",
+        "resource.script",
```

Comment:
The intention of the test was to make sure commas don't make it into the scrubbed description, and that is now trivially true for `resource.img` because the entire base name gets removed. But you are right, I will duplicate the tests instead.